server/middlewares/initialize.db.js

```diff
@@ -0,0 +1,28 @@
+import querryDb from '../helpers/v2/db.connector';
+import tables from '../storage/db';
+import queries from '../helpers/v2/db.querries';
+
+const admin = [
+    'premices@gmail.com',
+    'premices',
+    'tuverer',
+    'dddddd4U',
```

Comment:
consider using a general password eg `Password1`

Reply:
Noted

Reply:
Thank you